src/components/action-sheet/action-sheet.ios.scss

```diff
@@ -97,16 +97,15 @@ $action-sheet-ios-button-cancel-font-weight:            600 !default;
 
 .action-sheet-ios .action-sheet-group {
   @include border-radius($action-sheet-ios-border-radius);
+  @include margin(null, null, $action-sheet-ios-group-margin-bottom - 2, null);
```

Comment:
In theory, you are correct, and that is what I've done before. In practice no.
```scss
span {
    margin-top: 1px;
}
[dir=rtl] span {
    margin: 5px;
}
```

Gives the span `margin: 5px`, because of specificity.

```scss
[dir=rtl] span {
    margin-top: 1px;
}
[dir=rtl] span {
    margin: 5px;
}
```

Gives the span `margin: 1px 5px 5px 5px`, because of specificity. (which is the wanted behavior)

Don't worry, it gets super squashed at the end